deutsches-archaologisches-institut.csl

```diff
@@ -225,10 +220,10 @@
   <macro name="locators-article">
     <choose>
       <if type="article-newspaper">
-        <group prefix=", " delimiter=", ">
+        <group delimiter=", ">
           <group delimiter=" ">
-            <text variable="edition"/>
-            <text term="edition"/>
+            <text variable="edition" prefix=", "/>
```

Comment:
This is almost certainly wrong: 1) why would you have commas between the edition variable and the term? "Ausgabe, Hamburg"? But even if that's what you want, with the space delimiter and the spaces in prefix, you'd get "Ausgabe , Hamburg" which is definitely wrong.